.gitlab-ci.yml

```diff
@@ -143,3 +143,24 @@ publish_public_latest:
     IMG_SOURCES: $BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-amd64,$BUILD_DOCKER_REGISTRY/$PROJECTNAME:$CI_COMMIT_TAG-arm64
     IMG_DESTINATIONS: operator:latest
     IMG_SIGNING: "false"
+
+trigger_internal_image:
+  stage: release
+  rules:
+    - if: $CI_COMMIT_TAG
+      when: manual
+    - when: never
+  needs:
+    - intermediate_latest
```

Comment:
We should be able to remove the intermediate jobs now that we have Gitlab 13.9.

Also, the internal trigger should be automatically triggered, unlike to the public trigger.

Reply:
I removed for this job the `intermediate_latest ` need. 
But I will remove for the other jobs in a separated PR.